### PR TITLE
#121 - Replace 'full' -> 'Matrix'

### PR DESCRIPTION
**Pull request by schillic**
See #121.

### CODE REVIEW

File: test/unit_ExponentialMap.jl [hunk 1]
```diff
@@ -68,7 +68,7 @@ for N in [Float64, Float32]
     σ(sparsevec(d), emap)
 
     # check consistency with respect to explicit computation of the matrix exponential
-    svec_explicit = σ(d, expm(full(m)) * b)
+    svec_explicit = σ(d, expm(Matrix(m)) * b)
```
Comment (mforets) [inline test/unit_ExponentialMap.jl]: in julia 1.0 the matrix exponential function is also called `exp`. (should we use `Compat.expm` then?)


```julia
julia> expm(rand(2,2))
┌ Warning: `expm` is deprecated, use `exp` instead.
│   caller = top-level scope at none:0
└ @ Core none:0
2×2 Array{Float64,2}:
 2.35497  0.636294
 1.77983  1.93737 
```

Comment (mforets) [reply test/unit_ExponentialMap.jl]: or `Compat.exp`?

Comment (mforets) [reply test/unit_ExponentialMap.jl]: hmm i don't find it [here](https://github.com/JuliaLang/Compat.jl#new-functions-macros-and-methods), so i guess we have to touch our `compat.jl`..

Comment (schillic) [reply test/unit_ExponentialMap.jl]: Yes, this was supposed to be the next PR. It is not related to this one. One warning at a time :snail:

**Pull request merged**